### ISSUE TITLE
Master Thesis sections

### ISSUE THREAD

**Issue by Hexxers** [Zinoex/hyperverlet#1]
- [x] Summery
- [x] Abstract
- [x] Preface
- [x] Introduction
- [x] Theory
  - [x] Mechanics
    - [x] Newtonian
    - [x] Lagrangian
    - [x] Hamiltonian 
  - [x] Integrator
    - [x] [Truncation Error](https://www.researchgate.net/profile/Binbin-Li-2/publication/265908915_Error_Propagation_of_Verlet_Algorithm/links/542118150cf241a65a1e59db/Error-Propagation-of-Verlet-Algorithm.pdf) 
    - [x] Euler
    - [x] Heun
    - [x] RK4
    - [x] Velocity Verlet
    - [x] Forest-Ruth
- [x] Related Works
  - [x] ML for physical simulations
    - [x] [GNS](https://arxiv.org/pdf/2002.09405.pdf)
    - [x] [Learning Lagrangian Fluid Dynamics with Graph Neural Networks (FGN)](https://openreview.net/pdf?id=7WwYBADS3E_)
    - [x] [Machine Learning accelerated Computational Fluid Dynamics](https://arxiv.org/pdf/2102.01010.pdf)
    - [x] [Adaptive machine learning framework to accelerate ab initio molecular dynamics](https://onlinelibrary.wiley.com/doi/full/10.1002/qua.24836?casa_token=xxXjmMsCTrYAAAAA%3ANjh5vR10OHxY6P-UVuS0ssWvFJLjYJLbC7O0aGze3EIqx6TeTZklLN9dAFsC_J-uXXOcaBHNzhDukw)
    - [x] [JAX-MD](https://arxiv.org/abs/1912.04232)
    - [x] [TorchMD: A deep learning framework for molecular simulations](https://arxiv.org/abs/2012.12106)
  - [x] Neural ODE
    - [x] [Neural ODE](https://arxiv.org/abs/1806.07366)
    - [x] [Latent ODEs for Irregularly-Sampled Time Series](https://arxiv.org/abs/1907.03907)
    - [x] [Latent SDE](https://arxiv.org/pdf/2001.01328.pdf)
    - [x] [Graph NeuralODE](https://arxiv.org/pdf/1911.07532.pdf)
    - [x] [Learning differential equations that are easy to solve](https://arxiv.org/pdf/2007.04504.pdf)
  - [x] Symplectic Neural Networks
    - [x] [Variational Integrator Networks for Physically Structured Embeddings](https://arxiv.org/abs/1910.09349)
    - [x] [Symplectic Recurrent Neural Networks](https://arxiv.org/pdf/1909.13334.pdf)
    - [x] [Symplectic Neural Networks in Taylor Series Form for Hamiltonian Systems, about symmetric and symplectic Taylor series](https://arxiv.org/pdf/2005.04986.pdf)
    - [x] [Symplectic ODE-Net: Learning Hamiltonian Dynamics with Control](https://arxiv.org/pdf/1909.12077.pdf)
    - [x] [Hamiltonian Neural Networks for solving differential equations](https://arxiv.org/pdf/2001.11107.pdf)
  - [x] Physics inspired optimization
    - [x] [A variational perspective on accelerated methods in optimization](https://www.pnas.org/content/113/47/E7351)
    - [x] [Dynamical, Symplectic and Stochastic Perspectives on Gradient-Based Optimization](https://people.eecs.berkeley.edu/~jordan/papers/jordan-icm.pdf)
    - [x] [On Dissipative Symplectic Integration with Applications to Gradient-Based Optimization](https://arxiv.org/pdf/1802.03653.pdf)
  - [x] Deep integrators
    - [x] [Deep Euler Method](https://arxiv.org/abs/2003.09573)
    - [x] [Hyper euler / Hyper huen](https://arxiv.org/pdf/2007.09601.pdf)
- [x] Method
  - [x] Velocity verlet + hypersolver = hyperverlet
  - [x] Predictor-corrector architecture
  - [x] Residual training for VV
  - [x] Desirable properties of the NN
- [x] Experiment systems
  - [x] Pendulum
  - [x] Spring mass
  - [x] 3-body spring mass
- [x] Results
  - [x] Integrator comparison
    - [x] Spring mass
    - [x] Pendulum
    - [x] Three body spring mass
  - [x] Ablation study
    - [x] Underlying model type (for multibody systems)
    - [x] Underlying model size (show no overfitting)
    - [x] Apply NN between VV-steps or after
    - [x] Delta T power (including not multiplying by Delta T)
  - [x] Generalization study
    - [x] Train duration
    - [x] Variable parameters
    - [x] Out of distribution
- [x] Discussion
- [x] Conclusion
